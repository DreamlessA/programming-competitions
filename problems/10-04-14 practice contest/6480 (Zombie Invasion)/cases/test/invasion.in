10
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 77 0 0 99 0 0 0 40 0 0 0 17 0 1 13 10
43 15 77 23 58 28 91 66 31 42 26 13 3 62 15 91 18 68 33 72
29 83 93 43 62 92 7 94 81 0 86 17 87 53 61 97 46 82 20 44
98 68 6 56 82 24 74 35 65 77 38 22 8 33 36 33 87 61 8 38
90 38 64 59 70 95 84 45 7 13 44 77 64 15 29 55 99 9 94 59
81 33 93 12 38 30 89 12 27 82 28 0 27 22 52 86 19 50 16 95
62 78 38 6 20 49 76 7 5 27 65 31 74 100 48 36 40 84 44 77
66 10 58 47 33 72 38 49 71 33 54 20 17 74 4 49 33 4 67 50
