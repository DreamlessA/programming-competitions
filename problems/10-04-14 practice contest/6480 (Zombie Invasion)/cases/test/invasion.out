2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
5 1 0 0 1 1 0 1 1 0 0 1 0 0 1 1 1 0 0 0
79 0 1 0 0 1 1 1 1 0 1 1 1 0 1 1 1 0 1 0
107 0 0 1 0 1 0 0 1 1 1 0 0 1 1 1 1 1 0 0
148 0 1 0 0 0 1 0 0 1 0 0 1 0 1 1 0 0 1 0
141 1 0 0 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1
127 0 1 0 1 1 0 1 1 0 0 0 1 0 1 1 1 1 1 1
115 0 1 1 1 0 1 0 1 1 1 0 0 1 1 1 0 1 0 1
97 0 0 0 1 0 1 0 1 1 0 0 0 1 1 0 0 0 0 0
